proposals/0212-add-ability-to-reuse-a-syncfilename-for-a-putfile.md

```diff
@@ -37,6 +35,54 @@ When Core receives a `PutFile` it should send a notification to the HMI that a n
         Unique (during ignition cycle) id of the application. To be used in all RPCs sent by both HU system and SDL
         </description>
     </param>
+    
+    /* Existing parameters */
+    <param name="offset" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>
+        Optional offset in bytes for resuming partial data chunks
+        </description>
+    </param>
+    
+    <param name="length" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>
+        Optional length in bytes for resuming partial data chunks
+        </description>
+    </param>
+    
+    <param name="fileSize" type="Integer" minvalue="0" maxvalue="100000000000" mandatory="false">
+        <description>
+        Full Size of file. sends in first OnPutFile notification if file is splited into many PutFiles
```

Comment:
Update to:
Full Size of file. **Sends** in first OnPutFile notification **if the file is split** into many PutFiles

Or can it be:
Full Size of file is **sent** in first OnPutFile notification **if the file is split** into many PutFiles

Reply:
@jordynmackool I made the revisions. I created a new section for updating the documentation so the Core team knows they have to update the description for the `fileSize` parameter.

Reply:
@jordynmackool Do you want me to remove the above change and **not** make the update to documentation or do you want me to remove the above change but still make the update to the documentation?

Reply:
@NicoleYarroch make the noted change I commented above only. Then create an issue in the core repo about the documentation core update.